Sources/Public/Animation/LottieAnimation.swift

```diff
@@ -16,16 +16,25 @@ public enum CoordinateSpace: Int, Codable {
 
 // MARK: - Animation
 
-/// The `Animation` model is the top level model object in Lottie.
+@available(
+  *,
+  deprecated,
+  renamed: "LottieAnimation",
+  message: "`Lottie.Animation` has been renamed to `LottieAnimation`, to prevent conflicts with the `SwiftUI.Animation` type.")
```

Comment:
Maybe add a comment that this notice will be removed in Lottie 4.0?

Reply:
Good idea